src/form.js

```diff
@@ -154,6 +154,7 @@ define([
                     refs[mug.ufid] = {};
                 }
                 refs[mug.ufid][property || "."] = null;
+                that.internal = false;
```

Comment:
The meaning of this flag is sufficiently opaque that I think it would make sense to document how it affects instance metadata. It might make sense to put that in a docstring for the `InstanceMetadata` class.